responses/30-makefile-anatomy.md

```diff
@@ -1,82 +1,93 @@
+This file defines the relationships between different "targets" (see how the target `model_RMSEs_csv` is an input to the command that creates the target `plot_data`?), tells us where to find any functions that are used to build targets (see the `source` call that points you to `code.R`), and declares the package dependencies needed to build the different targets (see the `target_option_set` command that passes in a vector of packages). 
 
-Even though this is a simple example, there is some new syntax that may be confusing. We'll explain a few of these quickly:
- - `command` is a field that specifies what _function_ should be called in order to build each target.
- - `depends` is a field that explicity specifies a _dependency_ of a target. So when that dependency is considered "out of date" (more on that later), the target that lists that dependency in `depends` is also going to be "out of date". In this way, the "all" target can't be complete and up-to-date until `figure_1.png` is.
- - `all` is a special target that groups other targets. We'll cover group targets - like `all` - more later on.
- - `model_RMSEs.csv` shows up three times, why? `model_RMSEs.csv` is a file target that gets _created_ when the command `download_data` is run, and is an input to that same function in order to tell the function what file name to write the data to. Then `"model_RMSEs.csv"` shows up as an _input_ to another function, `process_data`, which looks like it reads in that file and changes it (or "processes" it) in some way.
+We'll briefly explain some of the functions and conventions used here. For more extensive explanations, visit [the `targets` documentation](https://books.ropensci.org/targets/walkthrough.html). 
 
+  - As you would with normal R scripts, put any `source` commands for loading R files and `library` commands for loading packages with functions used by subsequent code at the top of the file. 
+  - Declare each target by using the function `tar_target` and passing in a target name (`name` arg) and the expression to run to build the target (`command` arg).
+  - There are two types of targets - objects and files. If your target is a file, you need to add `format = "file"` to your `tar_targets` call and the command needs to return the filename of the new file.
+  - Setup the full pipeline by combining all targets into a single `list` object.
+  - There are 2 ways to define packages used by targets: 1) declare using the `packages` argument in `tar_option_set` in your makefile to specify packages used by all targets or 2) use the `packages` argument in individual `tar_target` functions for packages that may be special to specific targets.
```

Comment:
```suggestion
  - There are 2 ways to define packages used by targets: 1) declare using the `packages` argument in `tar_option_set()` in your makefile to specify packages used by all targets or 2) use the `packages` argument in individual `tar_target()` functions for packages that may be special to specific targets.
```
I find it easier to see what is a function if there are parentheses included for references to the function calls